.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "strict": 0
```

Comment:
Oh cool. Updated.

I originally tried to mimic https://github.com/codeclimate/codeclimate-eslint/blob/master/.eslintrc